### PR TITLE
Assignment 11: used tailwind css to style my exsisting application

### PR DESCRIPTION
**Pull request by nitishnivedan**
Assignment 10: used tailwind css to style my exsisting application

### CODE REVIEW

**Comment by nitishnivedan** [comment]
image
-------------------------------


image

**Comment by nikitaj-57** [comment]
Reviewed. Nicely Done!!

**Review by smriti-porwal — APPROVED** [review]

File: src/components/SearchComponent/index.js [hunk 1]
```diff
@@ -58,36 +59,46 @@ const SearchComponent = ({ setCurrentCards, teamInformation }) => {
...
+        </select>
+        <button className="bg-blue-700 h-12 w-28 font-bold m-5 rounded-xl text-white">
+          Search
+        </button>
+        <button
+          className={`h-12 w-44 font-bold m-5 rounded-xl border-black border-spacing-0  text-pink-500 ${
+            color === "purple" ? "bg-purple-50" : "bg-black"
+          }`}
+          onClick={() => setColor(color === "purple" ? "black" : "purple")}
+        >
```
Comment (roshantrivedi) [inline src/components/SearchComponent/index.js:101]: can move this function [setColor] out of button tag.

Comment (nitishnivedan) [reply src/components/SearchComponent/index.js:101]: moved

**Comment by iswarya15** [comment]
Reviewed. Nicely written with clean code

**Pull request merged**